src/styles/index.scss

```diff
@@ -1,3 +1,55 @@
 body {
   margin: 0;
 }
+
+$days: (
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sat: 6,
+  sun: 7,
+);
+$month-length: 31;
+$start-day: sun;
```

Comment:
The use of a map for days and variables for month length and start day is a good practice for flexibility and maintainability.